core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1620,8 +1620,8 @@ public void importTable(String tableName, Set<String> importDirs)
   }
 
   /**
-   * Prevent potential CRLF injection into logs from read in user data See
-   * https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS
+   * Prevent potential CRLF injection into logs from read in user data. See
+   * <a href="https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS">bug description</a>
```

Comment:
This also works, and doesn't require coming up with a good text description for the link:

```suggestion
   * Prevent potential CRLF injection into logs from read in user data.
   *
   * @see https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS
```

Reply:
It didn't like that...
<pre>
[WARNING] src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java:[1625] (javadoc) NonEmptyAtclauseDescription: Javadoc comment at column 16 has parse error. Details: mismatched input ':' expecting <EOF> while parsing JAVADOC
</pre>